html/changelogs/ambience_tweaks.yml

```diff
@@ -0,0 +1,6 @@
+author: SleepyGemmy
+
+delete-after: True
+
+changes:
+  - tweak: "Reworks and cleans up the ambience code."
```

Comment:
This is very vague. Elaborate on the fact that ship ambience is now its own SFX category.

Reply:
sir yes sir.